max7219/max7219.go

```diff
@@ -0,0 +1,440 @@
+// Copyright 2024 The Periph Authors. All rights reserved.
+// Use of this source code is governed under the Apache License, Version 2.0
+// that can be found in the LICENSE file.
+//
```

Comment:
Oops, you need to not put the `//` here otherwise the copyright will appear in the package documentation.

Reply:
Resolving so I can merge.